.github/workflows/pr.yml

```diff
@@ -33,14 +33,17 @@ jobs:
           # These are regex patterns auto-wrapped in `^ $`.
           scopes: |
             dapi
+            platform
```

Comment:
If it's platform (all or multiple packages) you just doesn't specify the scope.

Reply:
Well I see Drive+Drive-ABCI+DPP to be Platform, and very different from dashmate for example, often those 3 will be updated together. So I think it's a good thing to add and makes things more readable.

Reply:
Conceptually all packages in this repo are parts of the platform. Platform test suite, SDK, and DAPI...

"Drive+Drive-ABCI+DPP" for this we had a more general "drive", basically a blockchain runtime.

Reply:
Well that's just because of the history of teams that worked on the project a long time ago, Drive should be drive, not execution logic.